packages/flutter/test/material/scaffold_test.dart

```diff
@@ -147,6 +147,25 @@ void main() {
     expect(tester.binding.transientCallbackCount, greaterThan(0));
   });
 
+  testWidgets('Floating action button show/hide', (WidgetTester tester) async {
+    final GlobalKey one = new GlobalKey(debugLabel: 'one');
+    await tester.pumpWidget(new MaterialApp(home: new Scaffold(
+      floatingActionButton: new FloatingActionButton(
+        key: one,
+        onPressed: null,
+        child: const Text('1'),
+      ),
+    )));
+
+    expect(find.text('1').hitTestable(), findsOneWidget);
+    Scaffold.of(one.currentContext).hideFloatingActionButton();
+    await tester.pumpAndSettle();
+    expect(find.text('1').hitTestable(), findsNothing);
+Scaffold.of(one.currentContext).showFloatingActionButton().orCancel;
```

Comment:
nit: indentation

Why the `orCancel`?

Reply:
Doh.  That's not supposed ot be there.  I was having trouble getting the the right test to validate the FAB wasn't shown anymore.. will fix that.